sdk/modelsrepository/azure-iot-modelsrepository/src/main/java/com/azure/iot/modelsrepository/implementation/TempCustomType.java

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.iot.modelsrepository.implementation;
+
+import com.azure.iot.modelsrepository.implementation.models.FetchResult;
+
+import java.util.Map;
+
+/**
+ * This type is used to unify the expand operation return types in the recursive function and has no other use cases.
+ * Do not take any dependencies on this type.
+ */
+class TempCustomType {
```

Comment:
This should be given a better name even if it's in implementation package.

Reply:
Will think of a better name.